### PR TITLE
Pretty explanations

### PR DESCRIPTION
**Pull request by mmussomele**

### CODE REVIEW

File: docs/book/rest-api.md [hunk 1]
```diff
@@ -1643,9 +1643,13 @@ the following values:
...
 - **truth** - returns a partial query trace containing one path that leads to the overall query being successful.
 
+If the request asking for explanations sets the `pretty` parameter to `true`, instead of a JSON
+Array of [Trace Events](#trace-events), the server will instead return a human readable JSON string
+describing the explanation.
+
 ### <a name="trace-events"/>Trace Events
 
-When the `explain` query parameter is set to **full** or **truth** , the
+When the `explain` query parameter is set to **full** or **truth** and the pretty parameter is `false`, the
```
Comment (tsandall) [inline docs/book/rest-api.md]: Nit: Since false is the default, I'd omit this.

File: docs/book/rest-api.md [hunk 2]
```diff
@@ -1643,9 +1643,13 @@ the following values:
 - **full** - returns a full query trace containing every step in the query evaluation process.
 - **truth** - returns a partial query trace containing one path that leads to the overall query being successful.
 
+If the request asking for explanations sets the `pretty` parameter to `true`, instead of a JSON
```
Comment (tsandall) [inline docs/book/rest-api.md]: Nit: I'd reword this a bit...

```
By default, explanations are represented in a machine-friendly format. Set the `pretty` parameter to request a human-friendly format for debugging purposes.
```

**Comment by mmussomele** [comment]
Done

**Comment by mmussomele** [comment]
oh wait hold up

**Comment by mmussomele** [comment]
sorry, build was failing locally, fixed now 👍
